dev/import-beats/fields.go

```diff
@@ -141,9 +141,22 @@ func loadFieldsFile(path string) ([]fieldDefinition, error) {
 	if err != nil {
 		return nil, errors.Wrapf(err, "unmarshalling fields file failed (path: %s)", path)
 	}
+	fs = loadDefaultFieldValues(fs)
 	return fs, nil
 }
 
+func loadDefaultFieldValues(fs fieldDefinitionArray) fieldDefinitionArray {
+	var withDefaults fieldDefinitionArray
+	for _, f := range fs {
+		if f.Type == "" {
+			f.Type = "keyword"
```

Comment:
+1 on this. Less magic in the fields.yml and will make it easier to also validate the fields.yml files.